src/yall.mjs

```diff
@@ -63,7 +63,7 @@ export default function (options) {
 
   // If the current user agent is a known crawler, immediately load all media
   // for the elements yall is listening for and halt execution (good for SEO).
-  if (/(google|bing|yandex|duckduck)bot/i.test(navigator.userAgent)) {
+  if (/baidu|(?:google|bing|yandex|duckduck)bot/i.test(navigator.userAgent)) {
```

Comment:
Is this syntax widely supported? I can't recall if this is specific to ES6 or if has been around forever.

Reply:
yeah, forever i think. the fancy stuff is named capturing groups/backrefs, lookbehinds. i think positionl backreferences had some issues back in es3.

http://blog.stevenlevithan.com/archives/fixing-javascript-regexp